model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -218,11 +218,16 @@ def _create_input_dict(self, model_perf_analyzer_flags: Dict) -> Dict:
             return {}
 
     def _create_inference_load_list(self) -> List[Any]:
-        # The three possible inference loads are request rate, concurrency or periodic concurrency
-        # For LLM models periodic concurrency is used for non-LLM models
-        # concurrency is the default and will be used unless the user specifies
-        # request rate, either as a model parameter or a config option
-        if self._cli_config.is_llm_model():
+        # The possible inference loads are concurrency, request rate, periodic concurrency, or custom (request-intervals)
+        # - If custom is specified, it is used
+        # - For LLM models, periodic concurrency is used
+        # - For non-LLM models, concurrency is the default and will be used unless
+        #   the user specifies request rate (either as a model parameter or a config option)
+        #
+
+        if "request-intervals" in self._perf_analyzer_flags:
```

Comment:
Should make a list of custom inference load strings and have a method `is_custom_inference_load()` in perf_config.py